src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Props } from './Spinner.types'
+import spinner from './spinner.gif'
+
+const Spinner = ({ ...props }: Props): JSX.Element => {
```

Comment:
Can't it be shortened to this?
```suggestion
const Spinner = (props: Props): JSX.Element => {
```